Components/Images-Inputs/images-inputs.css

```diff
@@ -0,0 +1,92 @@
+@import url(/Styles/variables-and-fonts.css);
+
+
+
+/* <!-- Image Responsive and Round --> */
+
+.img-responsive {
+    display: block;
+    width: 100%;
+    height: auto;
+}
+
+
+.img-round {
+    width: 10rem;
+    height: 10rem;
+    border-radius: 50%;
+}
+
+
+/* <!-- Inputs default --> */
+
+.input-grp {
```

Comment:
Suggestion to separate styles for components.

Reply:
Okay thanks for the suggestion. Refactored it.